packages/webcomponents/src/components/checkout/saved-payment-method.tsx

```diff
@@ -22,30 +22,27 @@ export class SavedPaymentMethod {
     return { token: this.paymentMethodOption?.id };
   };
 
-  onPaymentMethodOptionClick = () => {
+  onPaymentMethodOptionClick = (e) => {
+    e.preventDefault();
     this.paymentMethodOptionSelected.emit(this.paymentMethodOption);
   };
 
   render() {
     return (
-      <Fragment>
-        <div class="payment-method">
-          <div
-            class="radio-list-item p-3"
-            onClick={() => this.onPaymentMethodOptionClick()}
-            part={radioListItem}
-          >
-            <form-control-radio
-              name="paymentMethodType"
-              id={this.paymentMethodOption?.id}
-              value={this.paymentMethodOption?.id}
-              checked={this.isSelected}
-              inputHandler={() => this.onPaymentMethodOptionClick()}
-              label={`${CardBrandLabels[this.paymentMethodOption?.brand]} *${this.paymentMethodOption?.acct_last_four}`}
-            />
-          </div>
+      <Host class="payment-method">
```

Comment:
Just curious about why we are using the `Host` element from Stencil here and not our `StyledHost`?

Reply:
It's because this component doesn't have shadow-dom, so it inherits the StyledHost styles from the parent.